pkg/math/testutils/median.go

```diff
@@ -38,6 +39,8 @@ func (m *MedianAggregator) SetProviderPrices(provider string, data types.Prices)
 	m.providerPrices[provider] = data
 }
 
+func (m *MedianAggregator) UpdateMarketMap(_ mmtypes.MarketMap) {}
```

Comment:
previously, we allowed a general interface of an aggregator to be passed into the oracle. in the orchestrator, it was explicitly the IndexAggregator. 

assuming we want to continue with the generalizable interface, i'm adding this method here. i'm not sure if it needs to do anything though.

Reply:
Yea this doesn't need to do anything. We don't use this aggregator and is mostly here just for testing purposes.